doc/user/metrics/service_monitor.md

```diff
@@ -0,0 +1,28 @@
+## Using the ServiceMonitor prometheus-operator CRD
+
+[prometheus-operator][prom-operator] an operator that creates, configures and manages Prometheus clusters atop Kubernetes.
+
+`ServiceMonitor` is a [CR][cr] of the prometheus-operator, which discovers the `Endpoints` in `Service` objects and configures Prometheus to monitor those `Pod`s. The `GenerateServiceMonitor` takes a `Service` object and generates a `ServiceMonitor` resource based on it.
+To add `Service` target discovery of your created monitoring `Service`s you can use the `metrics.CreateServiceMonitor()` helper function, which accepts the newly created `Service`.
+
+### Prerequisites:
+
+- [prometheus-operator][prom-quickstart] needs to be deployed already in the cluster.
+
+### Usage example:
+
+```go
+    import "github.com/operator-framework/operator-sdk/pkg/metrics"
+
+    // Pass the Service(s) to the helper function, which in turn returns the `ServiceMonitor` object.
+    serviceMonitors, err := metrics.CreateServiceMonitors(restConfig, "namespace", services)
+    if err != nil {
+        // handle error here
+    }
+```
+
+*Note:* Create one `ServiceMonitor` per application and per `Namespace`.
```

Comment:
Maybe expand on this a little bit.

Reply:
Moved this as a comment in code example so it hopefully makes more sense.